### DISCUSSION TITLE
KOBO 0560 freeze

### DISCUSSION THREAD

**Discussion by imisshifi** [koreader/koreader#12645] [Q&A]
Hello all, 
I just install the KOreader to my KOBO 0560 and when I want to get in to the reader, it's freeze and cannot to force restart/ factory reset.  What should I do?  Thank you.  I think the firmware version is "OCP-KOReader-v2024.07".  

https://github.com/user-attachments/assets/3260f017-db06-4658-997d-3073f9db0086

**Comment by benoit-pierre** [comment]
The discussion in #12633 might be relevant.

**Reply by imisshifi** [reply]
I've try lot of time and the screen still no any respond.  when I pin the factory reset button, even no any blue light.  

https://github.com/user-attachments/assets/de79d6b3-ce83-4972-ac3d-1d65e2b78c6a

**Comment by NiLuJe** [comment]
I have no idea what that device is, please use its actual name.
